make_unique doesn't compile with apple clang 13

I'm getting
```
  CXX      GDSIIgeom.lo
GDSIIgeom.cpp:147:26: error: no member named 'make_unique' in namespace 'std'
    auto vertices = std::make_unique<vector3[]>(num_vertices);
```
on `master` with macOS 11.5.2 and `Apple clang version 13.0.0 (clang-1300.0.29.3)`.

--- a/src/GDSIIgeom.cpp
+++ b/src/GDSIIgeom.cpp
@@ -144,7 +144,7 @@ geometric_object_list get_GDSII_prisms(material_type material, const char *GDSII
   for (int np = 0; np < num_prisms; np++) {
     dVec polygon = polygons[np];
     int num_vertices = polygon.size() / 2;
-    auto vertices = std::make_unique<vector3[]>(num_vertices);
+    std::unique_ptr<vector3[]> vertices(new vector3[num_vertices]);
     for (int nv = 0; nv < num_vertices; nv++) {
       vertices[nv].x = polygon[2 * nv + 0];
       vertices[nv].y = polygon[2 * nv + 1];
@@ -170,7 +170,7 @@ geometric_object get_GDSII_prism(material_type material, const char *GDSIIFile, 
   dVec polygon = get_polygon(GDSIIFile, Text, Layer);
 
   int num_vertices = polygon.size() / 2;
-  auto vertices = std::make_unique<vector3[]>(num_vertices);
+  std::unique_ptr<vector3[]> vertices(new vector3[num_vertices]);
   for (int nv = 0; nv < num_vertices; nv++) {
     vertices[nv].x = polygon[2 * nv + 0];
     vertices[nv].y = polygon[2 * nv + 1];

